### PR TITLE
Fix ProfileMenuBlockService and add tests

### PR DESCRIPTION
**Pull request by wbloszyk**
## Subject

I am targeting this branch, because PR #634 have little bug and need tests.

```markdown
### Fixed
- crash on Compile Error: Access level to Sonata\CustomerBundle\Block\ProfileMenuBlockService::getMenu() must be protected
```

### CODE REVIEW

File: src/CustomerBundle/Block/ProfileMenuBlockService.php [hunk 1]
```diff
@@ -58,7 +58,7 @@ public function configureSettings(OptionsResolver $resolver): void
         ]);
     }
 
-    private function getMenu(BlockContextInterface $blockContext): ItemInterface
+    protected function getMenu(BlockContextInterface $blockContext): ItemInterface
```
Comment (greg0ire) [inline src/CustomerBundle/Block/ProfileMenuBlockService.php]: Why is this needed?

Comment (wbloszyk) [reply src/CustomerBundle/Block/ProfileMenuBlockService.php]: getMenu is override method from ```Sonata\BlockBundle\Block\Service\MenuBlockService```

I got error:
```In ProfileMenuBlockService.php line 82:
                                                                                                                                                                                             
   Compile Error: Access level to 
   Sonata\CustomerBundle\Block\ProfileMenuBlockService::getMenu() must be protected (as in 
   class Sonata\BlockBundle\Block\Service\MenuBlockService) or weaker
```
See your [comment](https://github.com/sonata-project/ecommerce/pull/634#discussion_r362586924). Sorry, i did't check it. Anyway it is not diffrence between protected and private method when class is final.

Comment (wbloszyk) [reply src/CustomerBundle/Block/ProfileMenuBlockService.php]: Can you also retag ecommerce 3.2? It is big error. I didnt see it becouse sandbox cache correct version :(

Comment (greg0ire) [reply src/CustomerBundle/Block/ProfileMenuBlockService.php]: Please add a changelog note, I will create a patch version

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.2.1](https://github.com/sonata-project/ecommerce/compare/3.1.0...3.2.0) - 2020-01-03
+### Fixed
+- crash on Compile Error: Access level to Sonata\CustomerBundle\Block\ProfileMenuBlockService::getMenu() must be protected
```
Comment (greg0ire) [inline CHANGELOG.md]: I meant to just put this in your first message, please remove this diff
